demo/lit-element.html

```diff
@@ -11,38 +11,60 @@
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <title>lit-element demo</title>
   <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script type="module" src = "./ts-element.js"></script>
 </head>
 <body>
+    <my-element bar="5" whales="5" foobar="17">Hi</my-element>
+    <hr>
+    <ts-element message="Yo" more-info="person"></ts-element>
+
   <script type="module">
     import { LitElement, html } from '../lit-element.js';
 
+    class Inner extends LitElement {
+      render() {
+        return html`Hello world`;
+      }
+    }
+
+    customElements.define('x-inner', Inner);
+
     class MyElement extends LitElement {
 
       static get properties() {
         return {
-          foo: String,
-          bar: Number,
-          whales: Number
+          nug: {},
+          zot: {},
+          foo: {},
+          bar: {},
+          whales: {type: Number},
+          fooBar: {type: {fromAttribute: parseInt, toAttribute: (value) => value + '-attr'}, reflect: true}
         }
       }
 
+      get zot() { return this.getProperty('zot'); }
+
+      set zot(value) {
+        this.setProperty('zot', Number(value));
```

Comment:
Would be nice if the example explained exactly what you are trying to accomplish here. Like why is this useful?

Reply:
Probably something to cover in a getting started guide, but tweaked it a bit.